### DISCUSSION TITLE
in-editor help

### DISCUSSION THREAD

**Discussion by slonik-az** [helix-editor/helix#5094] [General]
Sorry for asking a possibly stupid question. I just started using helix after successfully compiling it from source on MacOS-12.6.1 (M1). But for the life of me I cannot figure out how to display help inside the editor. Usual suspects like `:h` or `:help` do not work. What am I missing?

**Comment by kirawi** [comment]
Not yet implemented: https://github.com/helix-editor/helix/issues/668

**Reply by slonik-az** [reply]
> Not yet implemented: #668

Great that it is already on the roadmap! For a new user ability to explore the editor and poke around cannot be underestimated. Besides, the competition (vim, neovim, emacs) all have it.
